scripts/release-tools/clusters.sh

```diff
@@ -212,6 +229,62 @@ create_qa_gke_cluster() {
   kubectl -n stackrox get pods
 }
 
+create_long_running_cluster() {
+
+  CLUSTER_NAME="$(get_cluster_name long-running)"
+  export CLUSTER_NAME
+
+  echo "cluster_name= $CLUSTER_NAME"
+  
+  status="$(check_cluster_status "$CLUSTER_NAME")"
+  
+  if does_cluster_exist "$CLUSTER_NAME"; then
+      echo "Unable to create cluster"
+  else
+      infractl create gke-default $CLUSTER_NAME --lifespan 168h --arg nodes=5 --wait --slack-me
+  fi
+  
+  # Set your local kubectl context to the remote cluster once the above completes successfully.
+  infractl get $CLUSTER_NAME --json | jq '.Connect' -r | bash
+
+  kubectl -n stackrox get pod
+  
+  
+  export MAIN_IMAGE_TAG=$(git describe --tags --abbrev=0) # Release version, e.g. 3.63.0-rc.2.
+  export API_ENDPOINT="localhost:8000"
+  
+  export STORAGE=pvc # Backing storage
+  export STORAGE_CLASS=faster # Runs on an SSD type
+  export STORAGE_SIZE=100 # 100G
+  export MONITORING_SUPPORT=true # Runs monitoring
+  export LOAD_BALANCER=lb
+  
+  toplevel_dir="$(git rev-parse --show-toplevel)"
+  "$toplevel_dir/deploy/k8s/central.sh" # Launches central
```

Comment:
This step failed for me, because docker daemon was not running

```
docker: Cannot connect to the Docker daemon at unix:///Users/prygiels/.colima/default/docker.sock. Is the docker daemon running?.
See 'docker run --help'.
```

Could we add a check for the daemon to be running at the beginning of this function?

(big plus for the current possibility to start docker, rerun the script, and not waste the cluster)

Reply:
I added a function check_docker_login. I was not sure about the best implementation, but it seems to work.

Reply:
Done